utils/airsim_manager.py

```diff
@@ -133,25 +128,17 @@ def get_cars_distance(self):
             np.array([[car2_position_and_speed["x"], car2_position_and_speed["y"]]])))
         return dist_c1_c2
 
-    def get_car1_state(self, logger):
+    def get_car1_state(self, logger=None):
         car1_position_and_speed = self.get_car_position_and_speed(self.config.CAR1_NAME)
-        car2_position_and_speed = self.get_car_position_and_speed(self.config.CAR2_NAME)
         car1_state = np.array([
             car1_position_and_speed["x"],
-            # car1_position_and_speed["y"],
             car1_position_and_speed["Vx"],
-            # car1_position_and_speed["Vy"],
-            # car2_position_and_speed["x"],
-            # car2_position_and_speed["y"],
-            # car2_position_and_speed["Vx"],
-            # car2_position_and_speed["Vy"],
-            # self.get_cars_distance()
         ])
 
-        # if self.config.LOG_CAR_STATES:
-        #     logger.log_state(car1_state, self.config.CAR1_NAME)
+        if logger is not None and self.config.LOG_CAR_STATES:
+            logger.log_state(car1_state, self.config.CAR1_NAME)
 
-        return car1_state
+        return car1_state  # מחזיר מערך NumPy
```

Comment:
comments should be in English

Reply:
Weird, at my PC the comment didn't appear..